openquakeplatform_taxtweb/utils/taxonomy.py

```diff
@@ -0,0 +1,131 @@
+from taxonomy_map import taxonomy_map
+
+class Ret(object):
+    def __init__(self, result=None, s=None):
+        self.result = result
+        self.s = s
+
```

Comment:
Maybe a variable name longer than `s`? It is not clear what it is its meaning.

Reply:
`s` mean "string", I'm following the same javascript nomenclature so I prefer continue to use it.